R/install.R

```diff
@@ -143,7 +143,7 @@ install <- function(packages = NULL,
     return(renv_pak_install(packages, libpaths, project))
   }
 
-  packages <- packages %||% renv_snapshot_dependencies(project, "implicit")
+  packages <- packages %||% renv_snapshot_dependencies(project, dev = TRUE)
```

Comment:
Do we need to document the fact that `install()` now respects the snapshot type? (The change probably makes sense)

Reply:
Oh yeah, good catch.